homeassistant/components/myq/cover.py

```diff
@@ -1,16 +1,22 @@
-"""Support for MyQ-Enabled Garage Doors."""
-import logging
+"""
+Support for MyQ-Enabled Garage Doors.
 
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/cover.myq/
```

Comment:
We don't use link in the file header anymore. Please revert the changes in the comment block.